index.md

```diff
@@ -8,6 +8,469 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+### C0102 Blacklisted name {#C0102}
+
+This error occurs when a variable name is listed in the black list. The black 
+list includes names like foo, bar, baz, toto, tata, tutu and so on. Students 
```

Comment:
"and so on" is not helpful here. Include a list of all blacklisted names that Pylint checks for. You can do this in markdown using 

```
- foo
- bar
- baz
- etc.
```